fe/src/main/java/org/apache/doris/optimizer/OptExpression.java

```diff
@@ -51,6 +51,12 @@ public OptExpression(OptOperator op, OptExpression... inputs) {
         this.inputs = Lists.newArrayList(inputs);
     }
 
+    public OptExpression(OptOperator op, OptExpression firstChild, OptExpression secondChild) {
```

Comment:
this `public OptExpression(OptOperator op, OptExpression... inputs) {` function can cover this funciton

Reply:
Ok, i will remove it.